go.mod

```diff
@@ -210,7 +225,7 @@ replace (
 	// opencontainers/runc patched a CVE in v1.1.5.
 	// Various dependencies both direct and indirect are yet to upgrade runc. All seem to work
 	// with v1.1.5 so we're forcing it with replace.
-	github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.5
+	//github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.5
```

Comment:
Delete this block?

Reply:
Oh, thanks. That's what happens when you leave something simple for several days :D.

Reply:
Fixed.